sourcegraph/sourcegraph
# JetBrains: Cody: Adding settings option to customize color of autocomplete hint

It resolves #56097 

## Test plan
* Go to cody settings page
* Check `Enable Cody autocomplete` checkbox
* Check `Enable custom autocomplete color` checkbox
* Pick the color that you want to use for cody autocomplete hint
* Press `OK`
* Trigger autocompletion


https://github.com/sourcegraph/sourcegraph/assets/9321940/cff8085d-e882-485a-a5d9-49e86dcbbf3c
